src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HeaderField.cs

```diff
@@ -30,5 +32,17 @@ public HeaderField(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value)
         public int Length => GetLength(Name.Length, Value.Length);
 
         public static int GetLength(int nameLength, int valueLenth) => nameLength + valueLenth + RfcOverhead;
+
+        public override string ToString()
+        {
+            if (Name?.Length > 0)
```

Comment:
When can Length be 0 here? I thought we just fixed it so it wouldn't be...

Reply:
This is a struct, so default value these will be `null`.

Reply:
The ctor does not prevent 0-length names; however we could certainly assert that instead of checking for null here. This member was added to make debugging easier so I went for the least-intrusive change.

Reply:
Yeah, I realize it's just for debugging and so doesn't really matter. But at the same time, when a class has a ToString implementation, I tend to look at how ToString is implemented in order to understand the possible/expected states of the class fields. So it's useful to have that represented here.